docker/test-entrypoint.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+set -e
+user=ipfs
+repo="$IPFS_PATH"
+echo "Version 0.4"
+
+if [ `id -u` -eq 0 ]; then
+    # ensure folders are writeable
+    su-exec "$user" test -w "$repo" || chown -R -- "$user" "$repo"
+    su-exec "$user" test -w "$IPFS_CLUSTER_PATH" || chown -R -- "$user" "$IPFS_CLUSTER_PATH"
+    # Make fifo with root priveleges
+    echo "making cluster-out-pipe fifo"
```

Comment:
Sorry, it was stupid of me to suggest you to do this since all logs go to stdout already... I must have been not very awake that day... can you get rid of it ? just do it how you did at the very beginning.